.asyncapi-tool

```diff
@@ -1,6 +1,8 @@
+---
 title: GitHub Action for Generator
 filters:
-    technology:
-        - AsyncAPI Generator
-    categories:
-        - github-actions
+  language: JavaScript
```

Comment:
no language, it is not a library, it is GH action, nobody can integrate it in their code